<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlayClass</class>
 <widget class="QWidget" name="XPlayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPlay</string>
  </property>
  <widget class="XVideoWidget" name="video">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openFile">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>630</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="XSlider" name="playPos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>1241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="isPlay">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>630</y>
     <width>81</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>播 放</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>xvideo_widget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>xslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>openFile</sender>
   <signal>clicked()</signal>
   <receiver>XPlayClass</receiver>
   <slot>OpenFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>759</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isPlay</sender>
   <signal>clicked()</signal>
   <receiver>XPlayClass</receiver>
   <slot>PlayOrPause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>782</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderPressed()</signal>
   <receiver>XPlayClass</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>825</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPos</sender>
   <signal>sliderReleased()</signal>
   <receiver>XPlayClass</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>694</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>757</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenFile()</slot>
  <slot>PlayOrPause()</slot>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
 </slots>
</ui>
